modules/nw-ovn-k-adminnetwork-policy-action-rules.adoc

```diff
@@ -31,9 +31,8 @@ spec:
     action: "Allow"
     from:
     - namespaces:
-        namespaceSelector:
-          matchLabels:
-            kubernetes.io/metadata.name: monitoring
+        matchLabels:
+          kubernetes.io/metadata.name: monitoring
 # ...
```

Comment:
I believe these indicate this is snippet of policy not complete policy.
@JoeAldinger

Reply:
Yes